plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopHang.java

```diff
@@ -39,8 +39,8 @@
     public void setUp()
             throws Exception
     {
-        File tempPath = createTempDir();
-        copyExecutable("hanging_atop.sh", tempPath);
+        Path tempPath = createTempDirectory(null);
```

Comment:
Please do add (in a separate commit)

```
        deleteRecursively(tempPath, ALLOW_INSECURE);
```

in the `tearDown` method in order to leave the testing environment in a clean state.

Reply:
I happened to find another undeleted temp directory in: 

- `io.trino.plugin.hive.s3.TestHiveS3Config#testExplicitPropertyMappings` 
- `io.trino.plugin.hive.rubix.TestRubixCaching#initializeRubix(io.trino.plugin.hive.rubix.RubixConfig, java.util.List<io.trino.spi.Node>)`
- `io.trino.plugin.hive.parquet.TestParquetDecimalScaling#getBasePath` 

These usecases are unrelated to this PR. Maybe another small PR would be better fit to host them.

Reply:
Let me raise this in another PR.